apps/server/src/modules/tool/external-tool/controller/dto/request/config/basic-tool-config.params.ts

```diff
@@ -1,14 +1,14 @@
-import { IsEnum, IsString } from 'class-validator';
 import { ApiProperty } from '@nestjs/swagger';
-import { ExternalToolConfigCreateParams } from './external-tool-config.params';
+import { IsEnum, IsUrl } from 'class-validator';
 import { ToolConfigType } from '../../../../../common/enum';
+import { ExternalToolConfigCreateParams } from './external-tool-config.params';
 
 export class BasicToolConfigParams extends ExternalToolConfigCreateParams {
 	@IsEnum(ToolConfigType)
 	@ApiProperty()
 	type!: ToolConfigType;
 
-	@IsString()
+	@IsUrl({ require_protocol: true, protocols: ['https'] })
```

Comment:
Which about the message? Currently the user only gets `400 - {"type":"API_VALIDATION_ERROR","title":"API Validation Error","message":"API validation failed, see validationErrors for details","code":400,"validationErrors":[{"field":["config","baseUrl"],"errors":["baseUrl must be a URL address"]}]}` if i put in http.

I would customize the message here